includes/wikia/GlobalFunctions.php

```diff
@@ -937,7 +937,11 @@ function wfTimeFormatAgo( $stamp, $hideCurrentYear = true ) {
 		$format = $wgLang->getDateFormatString( 'date', 'default' );
 		if ( $hideCurrentYear ) {
 			$format = trim( $format, ' ,yY' );
+
+			// For Japanese and Chinese the 年 character before the month number must also be stripped. See JPN-549
+			$format = str_replace('年', '', $format);
```

Comment:
Yes, but I think it is safe to say that str_replace with '年' will have no effect except for Japanese and Chinese.

Thanks for pointing out Messages*.php. This looks like the place to replace the default data format for Japanese and Chinese, so we can drop the (金) part from the date.

However, this fix is still required, since this is about the check for $hideCurrentYear being broken for Japanese and Chinese.